### PR TITLE
update DSL: digital_ocean content

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente <levente.bonczidai@hpe.com>

### CODE REVIEW

File: content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/create_droplet.sl [hunk 1]
```diff
@@ -74,20 +72,21 @@ flow:
...
   workflow:
     - execute_post_request:
         do:
           rest.http_client_post:
-            - url: "'https://api.digitalocean.com/v2/droplets'"
-            - auth_type: "'anonymous'"
-            - headers: "'Authorization: Bearer ' + token"
+            - url: 'https://api.digitalocean.com/v2/droplets'
+            - auth_type: 'anonymous'
+            - headers: "${'Authorization: Bearer ' + token}"
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/create_droplet.sl]: Why do we need the wrapping double quotes?

File: content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/delete_droplet.sl [hunk 2]
```diff
@@ -47,24 +47,24 @@ flow:
     - execute_delete_request:
         do:
           rest.http_client_delete:
-            - url: "'https://api.digitalocean.com/v2/droplets/' + droplet_id"
-            - auth_type: "'anonymous'"
-            - headers: "'Authorization: Bearer ' + token"
+            - url: ${'https://api.digitalocean.com/v2/droplets/' + droplet_id}
+            - auth_type: 'anonymous'
+            - headers: "${'Authorization: Bearer ' + token}"
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/delete_droplet.sl]: Again - wrapping double quotes

File: content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/list_all_droplets.sl [hunk 3]
```diff
@@ -51,33 +51,33 @@ flow:
     - execute_get_request:
         do:
           rest.http_client_get:
-            - url: "'https://api.digitalocean.com/v2/droplets'"
-            - auth_type: "'anonymous'"
-            - headers: "'Authorization: Bearer ' + token"
+            - url: ${'https://api.digitalocean.com/v2/droplets'}
+            - auth_type: 'anonymous'
+            - headers: "${'Authorization: Bearer ' + token}"
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/list_all_droplets.sl]: Again - double quotes

File: content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/retrieve_droplet_by_id.sl [hunk 4]
```diff
@@ -56,35 +56,35 @@ flow:
     - execute_get_request:
         do:
           rest.http_client_get:
-            - url: "'https://api.digitalocean.com/v2/droplets/' + str(droplet_id)"
-            - auth_type: "'anonymous'"
-            - headers: "'Authorization: Bearer ' + token"
+            - url: ${'https://api.digitalocean.com/v2/droplets/' + str(droplet_id)}
+            - auth_type: 'anonymous'
+            - headers: "${'Authorization: Bearer ' + token}"
```
Comment (oritstone) [inline content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/retrieve_droplet_by_id.sl]: Wrapping double quotes

**Comment by oritstone** [comment]
@Bonczidai small comments...
Is it tested yet?

File: content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/create_droplet.sl (hunk 1 above)
Comment (Bonczidai) [inline content/io/cloudslang/cloud_provider/digital_ocean/v2/droplets/create_droplet.sl]: error from Yaml:

```
ERROR:

mapping values are not allowed here
  in "<unicode string>", line 1, column 28:
    - headers: ${'Authorization: Bearer ' + token}
                               ^
```

**Comment by Bonczidai** [comment]
need to wait for dependencies before testing

**Comment by oritstone** [comment]
:+1:

**Pull request merged**